swarm/api/routes/runs_crud.py

```
@@ -0,0 +1,129 @@
+            start_step=request.start_step,
+        )
+
+        return RunStartResponse(
+            run_id=state["run_id"],
+            flow_id=state["flow_id"],
+            status=state["status"],
+            created_at=state["created_at"],
+            events_url=f"/api/runs/{state['run_id']}/events",
+        )
+
+    except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)